### ISSUE TITLE
Label the unlabeled UIElements in Package Manager UI for accessibility

### ISSUE THREAD

**Issue by harikmenon** [NuGet/Home#2745]
[From Saqib Shaikh]
There are two classes of low-hanging fruit: unlabelled buttons and inconsistent keyboard focus.

The unlabelled buttons should be straight forward to fix – we just need to set the AutomationElement.Name property, or override ToString() on the backing class.

In terms of keyboard focus, we need to make sure that you can tab and shift+tab to everything that is actionable. However, you shouldn’t be able to tab to a label, for example. Furthermore, you should be able to tab to a container (like a list) but not to individual items. I think there’s also some inconsistency in terms of how long it takes for the feed to load, so would need to figure out this scenario.

**Comment by rrelyea** [comment]
We'll target fixing the accessibility names for 3.4.5. The tabbing issue may not fix in this release.

**Comment by rrelyea** [comment]
let's discuss relative priorities. would love to see this catch the 3.4.5 release, but depends on your other work, etc...

**Comment by rohit21agrawal** [comment]
List of issues:
1) cannot narrate the title, texts inside, and buttons of License Acceptance dialog
2) cannot narrate the controls/texts from the File Conflict dialog
3) cannot narrate the install/uninstall progress dialog
4) cannot narrate the install/uninstall preview dialog
5) The scroll bar inbetween the list of package pane and the package details pane cannot be tabbed into
6) The scroll bar to the right of the package details pane cannot be tabbed into

**Comment by rohit21agrawal** [comment]
We have made some improvements in this space for the 3.5 release already - specifically, all the unlabeled UI in the Package manager window is now labeled. 
Going to keep this issue open, as we will continue to invest in this area in the future releases as well.
